loadbalancer.go

```diff
@@ -9,51 +9,47 @@ import (
 
 // LoadBalancerBackend represents a backend instance being load-balanced
 type LoadBalancerBackend struct {
-	InstanceID string `json:"instance_id"`
-	Protocol   string `json:"protocol"`
-	Port       int    `json:"port"`
+	IP              string `json:"ip"`
+	Protocol        string `json:"protocol,omitempty"`
+	SourcePort      int32  `json:"source_port"`
+	TargetPort      int32  `json:"target_port"`
+	HealthCheckPort int32  `json:"health_check_port,omitempty"`
 }
 
 // LoadBalancerBackendConfig is the configuration for creating backends
 type LoadBalancerBackendConfig struct {
-	InstanceID string `json:"instance_id"`
-	Protocol   string `json:"protocol"`
-	Port       int    `json:"port"`
+	IP              string `json:"ip"`
+	Protocol        string `json:"protocol,omitempty"`
+	SourcePort      int32  `json:"source_port"`
+	TargetPort      int32  `json:"target_port"`
+	HealthCheckPort int32  `json:"health_check_port,omitempty"`
 }
 
 // LoadBalancer represents a load balancer configuration within Civo
 type LoadBalancer struct {
```

Comment:
Should we have separate struct for "Create" , etc? We've seen this causing confusion when request and response params are in the same struct.

Reply:
`LoadBalancer` is the response struct and `LoadBalancerConfig` is used to create. We follow the same pattern for all objects.

Reply:
How do we differentiate between Create and Update request structs for `LoadBalancerConfig`?

Reply:
Create new struct for update request `LoadBalancerUpdateConfig`